user-guide/Troubleshooting/Procedures/Investigating_NATS_Issues.md

```diff
@@ -390,7 +390,34 @@ This will result in Agent 2 receiving different information about the same clust
 
 ## Check if port is already in use
 
-NATS uses port 9090 by default. If another program is already using this port, you will notice the following behavior:
+### Port 4222, 6222, and 8222
+
+NATS uses port 4222, 6222, and 8222. If another program is already bound to one of these ports, you will notice the following behavior:
+
+- DataMiner fails to start
+- NATS is stopped
+- The log file at *C:\Skyline DataMiner\NATS\nats-streaming-server\nats-server.log* can contain the following:
+
+  `Error listening on port: 0.0.0.0:4222, "listen tcp 0.0.0.0:4222: bind: Only one usage of each socket address (protocol/network address/port) is normally permitted."`
+
+In this case, the port may already be bound by one of the DataMiner processes. To fix this:
+
+1. Run the following command in a command prompt window: `netstat -aon | findstr 4222`.
```

Comment:
Would it also be possible to use 6222 or 8222 instead of 4222 here and on line 401? If yes, could you clarify this?